src/sidebars/favourites/tearout.js

```diff
@@ -21,8 +21,8 @@
                             };
 
                             if (tearout) {
-                                config.minWidth = tileWidth;
-                                config.minHeight = tileHeight;
+                                config.maxWidth = tileWidth;
```

Comment:
That's the size of the card being torn out, and using the values as a max made more sense and fixed an odd bug too. Perhaps it should be both min and max though.